server/routes/tests/delete.todo.test.js

```diff
@@ -0,0 +1,58 @@
+const request = require("supertest");
+const app = require("../../app");
+
+const expectedAllTodo = [
+    0: {
+        date: "21/03/2021",
+        description: "sleep",
+        entry_id: 1,
+        ticked: false,
+        time: "10:00",
+        title: "Shopping",
+    },
+    1: {
```

Comment:
Refer to previous comment for reason.
```suggestion
    {
```